tasks.py

```diff
@@ -262,6 +268,16 @@ def type_check(
         bin = "mypy"
 
     ge_pkgs = [f"great_expectations.{p}" for p in packages]
+
+    if check_stub_sources:
+        # see --help docs for explanation of this flag
+        for stub_file in GX_ROOT_DIR.glob("**/*.pyi"):
+            source_file = stub_file.with_name(  # TODO:py3.9 .with_stem()
+                f"{stub_file.name[:-1]}"
+            )
+            relative_path = source_file.relative_to(GX_ROOT_DIR.parent)
+            ge_pkgs.append(str(relative_path))
```

Comment:
This looks like it's adding `py` file. Is the `pyi` file already present and is it ok to have both? If we have both, do we need to run the check twice, once with the `--check-stub-sources` and once without, as you do in the pipeline yml files above?

Reply:
mypy errors if you ask it to check a `.py` and `.pyi` of the same name 😢